engine/src/main/java/io/camunda/zeebe/process/test/engine/EngineFactory.java

```diff
@@ -24,7 +25,7 @@
   public static InMemoryEngine create() {
     final int partitionId = 1;
     final int partitionCount = 1;
-    final int port = SocketUtil.getNextAddress().getPort();
+    final int port = 26500;
```

Comment:
This port gets used for the grpc server in the container. It used to be this value before we decided maybe we should make it dynamic for parallel testing. The problem when this is dynamic is that I wouldn't know what port it runs. This is an issue when I want to run the container and bind my container port to the internal port. By making it a static value I know how to bind the port.

I don't think it will cause any issues. This port is used inside of the container and since we control the container it feels like a safe assumption that nothing else will be running on this port. We could probably use an environment variable or something to make it configurable, but I wouldn't want to do this right now.